src/main/java/org/jabref/cli/Launcher.java

```diff
@@ -153,15 +134,17 @@ private static boolean handleMultipleAppInstances(String[] args, PreferencesServ
                 }
             } else {
                 // We are alone, so we start the server
-                Globals.REMOTE_LISTENER.openAndStart(new JabRefMessageHandler(), remotePreferences.getPort(), preferences);
+                Globals.REMOTE_LISTENER.openAndStart(new JabRefMessageHandler(), remotePreferences.getPort(),
```

Comment:
EasyBind.subscribe to remotePreferences.useRemoteServerProperty would make the use in Globals obsolete as the call to in NetwortTabViewModel::storeRemoteSettings could be abolished then...

Reply:
I agree, but let's keep this PR focused and leave further improvements to follow-ups!